api_types.ts

```diff
@@ -370,6 +424,22 @@ export interface SyncExecutionContext extends ExecutionContext {
 
 // A mapping exists in coda that allows these to show up in the UI.
 // If adding new values here, add them to that mapping and vice versa.
+/**
+ * Special "live" date range values that can be used as the {@link defaultValue}
+ * for a date array parameter.
+ *
+ * Date array parameters are meant to represent date ranges. A date range can
+ * be a fixed range, e.g. April 1, 2020 - May 15, 2020, or it can be a "live"
+ * range, like "last 30 days".
+ *
+ * At execution time, a date range will always be passed to a pack as an
+ * array of two specific dates, but for many use cases, it is necessary
```

Comment:
"always ... two specific dates" - What about in the non-builder case? Are you limited to only two dates in the list?